code/ZAS/Phoron.dm

```diff
@@ -87,8 +87,9 @@ obj/var/contaminated = 0
 	//Burn skin if exposed.
 	if(vsc.plc.SKIN_BURNS)
 		if(!pl_head_protected() || !pl_suit_protected())
-			burn_skin(0.75)
-			if(prob(20)) to_chat(src, "<span class='danger'>Your skin burns!</span>")
+			burn_skin(2)
+			if(prob(20))
+				to_chat(src, SPAN_DANGER("Your skin burns!"))
```

Comment:
It might be worth to up the probability of the message a bit so people notice it quicker if that happens (which is imho useful given the increased damage)

Reply:
When you're standing in phoron, a toxin indicator pops up, and your health doll starts showing the damage across the body, I think that's clear enough, don't really want to spam players with bold red text, but we can see how players react to it on live, I think.